packages/components/src/date-time/README.md

```diff
@@ -71,3 +71,10 @@ A callback invoked when selecting the previous/next month in the date picker. Th
 
 -   Type: `Function`
 -   Required: No
+
+### events
+
+List of events to show in the date picker. Each event will appear as a dot on the day of the event.
+
+-   Type: `Array`
+-   Required: No
```

Comment:
There's a [preferred props format](https://github.com/WordPress/gutenberg/pull/32392#discussion_r679028969) for newer components. Now might be a good opportunity to update this `README.md` to meet that.
```suggestion
### events: `Array`

List of events to show in the date picker. Each event will appear as a dot on the day of the event.

-   Required: No
```

The primary difference is moving the `type` inline to the prop name heading.